crates/aptos/src/move_tool/mod.rs

```diff
@@ -304,7 +304,7 @@ impl CliCommand<&'static str> for ProvePackage {
 
         match result {
             Ok(_) => Ok("Success"),
-            Err(_) => Err(CliError::MoveProverError),
+            Err(err) => Err(CliError::MoveProverError(err.to_string())),
```

Comment:
To print an anyhow error with the chain of causes, `format!("{:#}", err)` should be used. `:#` means alternate form of string representation, very cryptic... would prefer `err.chain_to_string()`.